src/kinetics/PlogRate.cpp

```diff
@@ -120,6 +116,14 @@ void PlogRate::setRates(const std::multimap<double, ArrheniusRate>& rates)
         j++;
         rates_.push_back(rate.second);
     }
+    if (!m_valid) {
+        // ensure that reaction rate can be evaluated (but returns NaN)
+        rates_.reserve(2);
+        pressures_[std::log(1.e-7)] = {0, 1};
+        rates_.push_back(ArrheniusRate());
+        pressures_[std::log(1.e14)] = {1, 2};
```

Comment:
Why these pressures? Would `TINY` and `HUGE` be better? You can probably also drop the log, right? Just -7 and 14 are the results of those calculations?

Reply:
`TINY` and `HUGE` are defined in very different contexts. Using `Cantera::Tiny` (1e-20) and `1. / Cantera::Tiny` instead.